tests/utils.go

```diff
@@ -3473,6 +3473,24 @@ func SkipIfPrometheusRuleIsNotEnabled(virtClient kubecli.KubevirtClient) {
 	}
 }
 
+func SkipIfSingleReplica(virtClient kubecli.KubevirtClient) {
+	kv := util2.GetCurrentKv(virtClient)
+	if kv.Spec.Infra != nil && kv.Spec.Infra.Replicas != nil {
+		if *(kv.Spec.Infra.Replicas) == 1 {
+			Skip("Skip multi-replica test on single-replica deployments")
+		}
+	}
+}
+
+func SkipIfMultiReplica(virtClient kubecli.KubevirtClient) {
+	kv := util2.GetCurrentKv(virtClient)
+	if kv.Spec.Infra != nil {
```

Comment:
I think we should skip even if `kv.Spec.Infra == nil ` -> `replicas ==2`

Reply:
Nice catch, thanks, fixed.